ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -203,6 +206,17 @@ class ManagementResources(processCounter: ProcessCounter,
             }
           }
         }
+      } ~
+      path("processManagement" / "customAction" / Segment) { processName =>
+        (post & processId(processName) & entity(as[CustomActionRequest])) { (process, req) =>
+          complete {
+            (managementActor ? (req.toEngineRequest(process.name), process.id, user))
+              .mapTo[Either[CustomActionError, CustomActionResult]]
+              .map(CustomActionResponse(_))
+              .flatMap(Marshal(_).to[MessageEntity])
+              .map(en => HttpResponse(entity = en))
```

Comment:
Do we always return 200?

Reply:
yes, it's always 200,
in case of action failure we return { isSuccess: false, ...}